src/draw/shapes.typ

```diff
@@ -490,20 +491,25 @@
 /// - close (bool): If true, the line-strip gets closed to form a polygon
 /// - name (none,str):
 ///
-/// ## Styling 
+/// ## Styling
 /// *Root:* `line`
 ///
 /// Supports mark styling.
-/// 
-/// ## Anchors
-/// Supports path anchors.
+///
+/// = Anchors
+///   Supports path anchors.
+///   / centroid: The centroid anchor is calculated for _closed non self-intersecting_ polygons if all vertices share the same z value.
```

Comment:
I need to write a styling guide for the new docs but heres a quick run down:
- Use `##` for the anchors heading instead of `=`
- Use a list for anchor names and wrap the name in double `*` so `- **centroid**: `
- wrap in single `*` to make italics not underscores